backend/substrapp/tasks/tasks.py

```diff
@@ -137,34 +142,43 @@ def find_training_step_tuple_from_key(tuple_key):
         f'Key {tuple_key}: no tuple found for training step: model: {metadata}')
 
 
-def get_and_put_model_content(tuple_type, tuple_key, tuple_, out_model, model_dst_path):
+def get_and_put_model_content(tuple_type, hash_key, tuple_, out_model, model_dst_path):
     """Get out model content."""
     owner = tuple_get_owner(tuple_type, tuple_)
     return get_and_put_asset_content(
         out_model['storageAddress'],
         owner,
         out_model['hash'],
         content_dst_path=model_dst_path,
-        salt=tuple_key
+        hash_key=hash_key
```

Comment:
I don't understand the change from `salt` to hash_key for the function arguments, can you explain :) ?

Reply:
This is to be consistent with the names of the arguments of the `get_hash` function. It doesn't have a concept of "salt", only of a key, so I fixed the inconsistency.